reference/generators/cmake_find_package.rst

```diff
@@ -55,8 +55,56 @@ utilities exported by the package available for consumers just by setting `find_
 Moreover, this also adjusts `CMAKE_MODULE_PATH` and `CMAKE_PREFIX_PATH` to the values declared by the package in ``cpp_info.buildirs``, so
 modules in those directories can be found.
 
-Target in Find<PKG-NAME>.cmake
-------------------------------
+Components
+++++++++++
+
+If a recipe defines different components using ``self.cpp_info.components``, the variables declared in *Find<PKG-NAME>.cmake* will be:
+
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| NAME                               | VALUE                                                                                               |
++====================================+=====================================================================================================+
+| <PKG-NAME>_FOUND                   | Set to 1                                                                                            |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <PKG-NAME>_VERSION                 | Package version                                                                                     |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <PKG-NAME>_COMPONENTS              | List of components of the package                                                                   |
++------------------------------------+-----------------------------------------------------------------------------------------------------+
+| <COMP-NAME>_INCLUDE_DIRS           | Containing all the include directories of the component                                             |
```

Comment:
I'm not sure if these variables should have the package as prefix: `<PKG-NAME>_<COMP-NAME>_INCLUDE_DIRS`, otherwise there can be collisions between packages with components with the same name.

Maybe we should have a look to existing CMake's modules and see if (and how) they expose information about components.